### DISCUSSION TITLE
Movable spinners!

### DISCUSSION THREAD

**Discussion by LacticWhale** [ppy/osu#19862] [Ideas] [answered]
Imagine spinning in different parts of the screen.

Possible features:
- Ability to move spinner position in editor.
- Mods can affect spinner position. 
  - Imagine following your spinner with repel mod on.
- Additional mod for random spinner position. (It will be fun. Trust me).
  - [Ievan Polkka](https://osu.ppy.sh/beatmapsets/36323) will actually be fun to play.

**Comment by peppy** [comment, marked as answer]
Already supported internally, and will be added properly in the future.

**Reply by LacticWhale** [reply]
This will be my most anticipated update.

**Comment by iSimpFor** [comment]
maybe the spinner can move as if on a slider path, making you deal with a moving spinner

**Reply by LacticWhale** [reply]
Sounds good
